src/org/opendatakit/common/cli/Args.java

```diff
@@ -90,18 +90,6 @@ public boolean has(Param<?> key) {
         .orElseThrow(() -> new IllegalArgumentException("No param -" + key.shortCode + " has been declared"));
   }
 
-  /**
-   * <p>Shortcut to <code>getOptional(key).orElse(null)</code> to avoid handling explicit
-   * null values
-   *
-   * @param key the {@link Param} key of the value to be retrieved
-   * @param <T> type of the value to be retrieved
-   * @return the value contained in the given <code>key</code> or <code>null</code> if the key is not contained
-   */
-  public <T> T getOrNull(Param<T> key) {
```

Comment:
new/revisit

This class is an internal library. I think having non-used methods is not so important.